lms/static/js/spec/main_requirejs_coffee.js

```diff
@@ -17,6 +18,22 @@
         },
         "afontgarde": {
             exports: "AFontGarde"
+        },
+        shim: {
+            'coffee/src/instructor_dashboard/send_email': {
+                exports: 'coffee/src/instructor_dashboard/send_email',
+                deps: ['jquery', 'underscore'],
+                init: function() {
+                    require([
+                        'edx-ui-toolkit/js/utils/html-utils',
+                        'edx-ui-toolkit/js/utils/string-utils'
+                    ], function (HtmlUtils, StringUtils) {
+                        window.edx = edx || {};
+                        window.edx.HtmlUtils = HtmlUtils;
+                        window.edx.StringUtils = StringUtils;
+                    });
+                }
+            }
```

Comment:
Hey @andy-armstrong, I'm running into some more troubles getting `edx.StringUtils` loaded properly in tests. `coffee/src/instructor_dashboard/send_email.coffee` is run with `lms-coffee`, so the previous use of `js/spec/main.js` didn't help. Is this the right file to be making this modification for `lms-coffee` tests? It doesn't seem to be working for me.